charts/k8ssandra/values.yaml

```diff
@@ -134,15 +134,29 @@ backupRestore:
     # Set to true to use the same bucket across multiple clusters.
     multiTenant: false
 
-    # Accepted values are s3 and gcs
-    storage: s3
+    # Accepted values are s3, s3_compatible (S3 compatible storages) and gcs
+    storage: s3_compatible
+
+    # Optional properties for storage
+    storage_properties:
```

Comment:
Are there any properties that are shared between s3 and s3_compatible? If not I wonder if might be clearer to separate them by storage type.

Reply:
region